plugins/htpasswd/package.json

```diff
@@ -34,29 +36,20 @@
     "unix-crypt-td-js": "1.0.0"
   },
   "devDependencies": {
-    "@commitlint/cli": "7.5.2",
-    "@commitlint/config-conventional": "7.5.0",
-    "@octokit/rest": "16.23.2",
-    "@types/bcryptjs": "2.4.2",
-    "@types/http-errors": "1.6.1",
-    "@types/jest": "24.0.11",
-    "@types/node": "11.12.1",
-    "@verdaccio/babel-preset": "0.1.0",
-    "@verdaccio/types": "5.0.0-beta.4",
-    "@verdaccio/eslint-config": "0.0.1",
-    "codecov": "3.2.0",
-    "cross-env": "5.2.0",
-    "eslint": "5.15.3",
-    "get-stdin": "6.0.0",
-    "husky": "0.14.3",
-    "in-publish": "2.0.0",
-    "jest": "24.5.0",
-    "lint-staged": "7.2.0",
-    "standard-version": "5.0.2",
-    "typescript": "3.2.4"
+    "@commitlint/config-conventional": "^7.5.0",
```

Comment:
We don’t need commitlint settings in a plugin package, it’s defined in the root :wink: